src/components/editor/property/PropertyComponent.jsx

```diff
@@ -8,20 +8,18 @@ import NestedResource from "./NestedResource"
 import ReadOnlyInputLiteralOrURI from "../inputs/ReadOnlyInputLiteralOrURI"
 import Alert from "../../Alert"
 import { displayResourceValidations } from "selectors/errors"
-import { selectNormSubject } from "selectors/resources"
+import { selectUri } from "selectors/resources"
 
 // Decides how to render this property.
 const PropertyComponent = ({ property, propertyTemplate, readOnly }) => {
-  const resource = useSelector((state) =>
-    selectNormSubject(state, property.rootSubjectKey)
-  )
+  const uri = useSelector((state) => selectUri(state, property.rootSubjectKey))
```

Comment:
Only need the URI, so only get the URI.

Reply:
only looking at the diff here - is there any other use of uri in the file (i.e. should it be named propertyUri) ?